src/PageContent/Calendar/ContactContent.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import phone from '../../icons/phone.svg';
+
+class ContactContent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            edit: false,
+            contact: {},
+            memberList: []
+        };
+        if (this.props.accountType === AccountType.DONATING_AGENCY_MEMBER) {
+            this.state.contact = this.props.delivery.donatingAgency.contact;
+            this.state.memberList = this.props.delivery.donatingAgency.contact.memberList;
+        } else if (this.props.accountType === AccountType.RECEIVING_AGENCY) {
+            this.state.contact = this.props.delivery.receivingAgency.contact;
+        }
+
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleChangeDA = this.handleChangeDA.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+        this.setState({
+            name: e.target.name.value,
+            phone: e.target.phone.value,
+            edit: false
+        });
+    }
+
+    handleChangeDA(e) {
+        e.preventDefault();
+        let index = e.target.primaryContact.value;
+        let memberName = this.state.memberList[index].name;
+        let memberPhone = this.state.memberList[index].phone;
+        this.setState({
+            name: memberName,
+            phone: memberPhone,
+            edit: false
+        });
+    }
+
+    render() {
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={phone} alt="volunteer" />
+                <div className="content-wrapper content-wrapper-description">
+                    {this.props.accountType === AccountType.RECEIVING_AGENCY ? (
+                        <h1 className="section-header">
+                            Primary Contact for Delivery
+                        </h1>
+                    ) : this.props.accountType ===
+                    AccountType.DONATING_AGENCY_MEMBER ? (
+                            <h1 className="section-header">
+                            Primary Contact for Pick Up
+                            </h1>
+                        ) : null}
+                    {!this.state.edit ? (
+                        <div>
+                            {!this.state.name ? (
```

Comment:
there is no need to have this check if you just always use/modify `this.state.contact`